extensions/wikia/AdEngine/js/provider/gpt/adDetect.js

```diff
@@ -302,6 +307,29 @@ define('ext.wikia.adEngine.provider.gpt.adDetect', [
 
 		if (expectAsyncHop || expectAsyncHopWithSlotName) {
 			if (shouldPollForSuccess || expectAsyncSuccess || expectAsyncSuccessWithSlotName) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Change this condition so that it does not always evaluate to "true". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2583)